### PR TITLE
Add "pressure" to attribute

### PR DESCRIPTION
**Pull request by renpj**

### CODE REVIEW

**Comment by TDI-tenderholt** [comment]
Overall, it looks fine. Can you write a test for this new attribute? Should be in ```testvib.py```.

Also, a few ideas for improvement:

* Can you also implement this attribute in the other parsers?
* We prefer something like ```line[x:y] == 'Pressure'``` instead of ```find``` (see the check for Temperature just above).
* Since Temperature and Pressure appear in the same line, can you combine so there isn't a redundant ```line.split```()?

**Comment by renpj** [comment]
@TDI-tenderholt 

- I've combined temperature and pressure into same line.
- I don't familiar with other program, so it is difficult to implement to other parsers.
- Are you sure that test for this attribute should put in testvib.py? It seem all about IR and Raman infomations.

**Comment by TDI-tenderholt** [comment]
Temperature and pressure only seem relevant in the vibrational calculations. For example, there's a ```testtemperature``` function in ```testvib.py```, but I now see that it's only in the custom ```QChemIRTest``` class.

@langner & @berquist : Do you think we should go ahead and merge this PR or add support to other parsers + implement a general test function? I think we can easily get coverage for ADF, DALTON, GAMESS-UK, and ORCA:

```
~/source/cclib/data]$ grep -i pressure */basic*/*
ADF/basicADF2007.01/dvb_ir.adfout: Pressure:                  1.000000 atm.
ADF/basicADF2013.01/dvb_ir.adfout: Pressure:                  1.000000 atm.
DALTON/basicDALTON-2013/dvb_ir.out: Qtran is evaluated per mol at 1 atm. pressure.
DALTON/basicDALTON-2013/dvb_raman.out: Qtran is evaluated per mol at 1 atm. pressure.
DALTON/basicDALTON-2015/dvb_ir.out: Qtran is evaluated per mol at 1 atm. pressure.
DALTON/basicDALTON-2015/dvb_raman.out: Qtran is evaluated per mol at 1 atm. pressure.
GAMESS-UK/basicGAMESS-UK7.0/dvb_ir.out: pressure      1.00000 atm
GAMESS-UK/basicGAMESS-UK7.0/dvb_ir.out:                     internal     constant pressure       entropy
GAMESS-UK/basicGAMESS-UK7.0/dvb_ir.out:                     internal     constant pressure       entropy
GAMESS-UK/basicGAMESS-UK7.0/dvb_raman.out: pressure      1.00000 atm
GAMESS-UK/basicGAMESS-UK7.0/dvb_raman.out:                     internal     constant pressure       entropy
GAMESS-UK/basicGAMESS-UK7.0/dvb_raman.out:                     internal     constant pressure       entropy
GAMESS-UK/basicGAMESS-UK8.0/dvb_ir.out: pressure      1.00000 atm
GAMESS-UK/basicGAMESS-UK8.0/dvb_ir.out:                     internal     constant pressure       entropy
GAMESS-UK/basicGAMESS-UK8.0/dvb_ir.out:                     internal     constant pressure       entropy
GAMESS-UK/basicGAMESS-UK8.0/dvb_raman.out: pressure      1.00000 atm
GAMESS-UK/basicGAMESS-UK8.0/dvb_raman.out:                     internal     constant pressure       entropy
GAMESS-UK/basicGAMESS-UK8.0/dvb_raman.out:                     internal     constant pressure       entropy
Gaussian/basicGaussian03/dvb_ir.out: Temperature   298.150 Kelvin.  Pressure   1.00000 Atm.
Gaussian/basicGaussian03/dvb_raman.out: Temperature   298.150 Kelvin.  Pressure   1.00000 Atm.
Gaussian/basicGaussian09/dvb_ir.out: Temperature   298.150 Kelvin.  Pressure   1.00000 Atm.
Gaussian/basicGaussian09/dvb_raman.out: Temperature   298.150 Kelvin.  Pressure   1.00000 Atm.
ORCA/basicORCA2.9/dvb_ir.out:Pressure            ... 1.00 atm
ORCA/basicORCA2.9/dvb_raman.out:Pressure            ... 1.00 atm
ORCA/basicORCA3.0/dvb_ir.out:Pressure            ... 1.00 atm
ORCA/basicORCA3.0/dvb_raman.out:Pressure            ... 1.00 atm
```

File: src/cclib/parser/data.py [hunk 1]
```diff
@@ -61,7 +61,8 @@ class ccData(object):
         nooccnos -- natural orbital occupation numbers (array[1])
         optdone -- flags whether an optimization has converged (Boolean)
         optstatus -- optimization status for each set of atomic coordinates (array[1])
-        polarizabilities -- (dipole) polarizabilities, static or dynamic (list of arrays[2])
+        polarizabilities -- (dipole) polarizabilities, static or dynamic (list of arrays[2])
```
Comment (langner) [inline src/cclib/parser/data.py]: Why is this line marked as changed?

File: src/cclib/parser/data.py [hunk 2]
```diff
@@ -131,7 +132,8 @@ class ccData(object):
        "nooccnos":         Attribute(numpy.ndarray,    'TBD',                         'N/A'),
        "optdone":          Attribute(bool,             'done',                        'optimization'),
        "optstatus":        Attribute(numpy.ndarray,    'status',                      'optimization'),
-       "polarizabilities": Attribute(list,             'polarizabilities',            'N/A'),
+       "polarizabilities": Attribute(list,             'polarizabilities',            'N/A'),
```
Comment (langner) [inline src/cclib/parser/data.py]: Why is this line marked as changed?

**Comment by langner** [comment]
I think we can merge in it's current form an extend the attribute later.

However, @renpj could you squash you commits to a single one? Bonus points for adding the unit test Adam asked about.

File: src/cclib/parser/data.py (hunk 2 above)
Comment (renpj) [inline src/cclib/parser/data.py]: This line shouldn't be marked. I didn't change this line, just omit this change.

Comment (langner) [inline src/cclib/parser/data.py]: It looks like the line is marked because you added a carriage return on it.

**Comment by renpj** [comment]
@langner I have squashed my commit.

**Comment by berquist** [comment]
I think we can slowly add support for other packages.

**Comment by langner** [comment]
> I think we can slowly add support for other packages.

Agree - we can merge this and deal with tests later, too. See #370 also for enforcing consistent line endings.

Thanks @renpj!

**Pull request merged**